java/build/src/main/java/sleeper/build/github/api/containers/DeleteGHCRVersions.java

```diff
@@ -76,17 +77,19 @@ public static Builder withApi(GitHubApi api) {
     }
 
     public void deleteVersions() {
-        LOGGER.info("Deleting versions for {}/{}, ignoring {} and keeping {}",
-                organization, packageName, ignoreTags, keepMostRecent);
-        List<GitHubPackageVersionResponse> all = getAllVersions();
-        List<GitHubPackageVersionResponse> toDelete = getVersionsToDelete(all).collect(Collectors.toList());
-        LOGGER.info("Deleting {} of {} versions", toDelete.size(), all.size());
-        toDelete.forEach(this::deleteVersion);
+        LOGGER.info("Deleting versions for {} in org {}, ignoring {} and keeping {}",
+                organization, packageNames, ignoreTags, keepMostRecent);
```

Comment:
These parameters to the logger look like they're in the wrong order. It looks like it should be 
`packageNames, organization, ignoreTags, keepMostRecent`

Reply:
Fixed, good spot.